### PR TITLE
docs: custom aliases

### PR DESCRIPTION
**Pull request by zirain**
use custom alias.html to make to `/tasks/quickstart` redirect to `/<current_version>/tasks/quickstart`

### CODE REVIEW

File: site/layouts/alias.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{{- if (strings.Contains .Permalink "v1.0.2") -}}
```
Comment (zirain) [inline site/layouts/alias.html:1]: didn't find a good way to make `v1.0.2` read from some place.

Comment (arkodg) [reply site/layouts/alias.html:1]: cc @Xunzhuo

File: site/content/en/latest/install/gateway-addons-helm-api.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
-+++
```
Comment (zirain) [inline site/content/en/latest/install/gateway-addons-helm-api.md:1]: we should use same style `---` in every page.

**Comment by arkodg** [comment]
thanks @zirain !
quick q - how this is impact our release process 
cc @guydc @Xunzhuo

File: site/content/en/boilerplates/o11y_prerequisites.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 ---
-Follow the steps from the [Quickstart](../quickstart) to install Envoy Gateway and the example manifest.
+Follow the steps from the [Quickstart](/tasks/quickstart) to install Envoy Gateway and the example manifest.
```
Comment (zirain) [inline site/content/en/boilerplates/o11y_prerequisites.md:3]: this should be versioned, but it's fine now.

**Comment by zirain** [comment]
> thanks @zirain ! quick q - how this is impact our release process cc @guydc @Xunzhuo

should be same as before except https://github.com/envoyproxy/gateway/pull/3654#discussion_r1649848623

**Comment by arkodg** [comment]
@zirain can you confirm if a URL like `https://gateway.envoyproxy.io/tasks/observability/grafana-integration/` will redirect to `https://gateway.envoyproxy.io/v1.0.2/tasks/observability/grafana-integration/` ?

this will impact the URL in the browser, so anyone adding these links in other places will have to manually delete the version from the path

**Comment by arkodg** [comment]
@zirain one issue with the redirect approach is, it negatively impacts SEO, so ideally prefer if something like `https://gateway.envoyproxy.io/tasks/observability/grafana-integration/` can return the current docs (v1.0.2 in this case) w/o 
changing the URL 

https://developers.google.com/search/docs/crawling-indexing/301-redirects?hl=en&sjid=413640029458435414-NC&visit_id=638550359061002202-3950558963&rd=1 highlights that for permanent redirects, the target URL is treated as the strong signal

**Comment by zirain** [comment]
> @zirain can you confirm if a URL like `https://gateway.envoyproxy.io/tasks/observability/grafana-integration/` will redirect to `https://gateway.envoyproxy.io/v1.0.2/tasks/observability/grafana-integration/` ?
> 
> this will impact the URL in the browser, so anyone adding these links in other places will have to manually delete the version from the path

```
Request URL:
http://localhost:1313/tasks/observability/proxy-observability/
Request Method:
GET
Status Code:
200 OK
Remote Address:
[::1]:1313
Referrer Policy:
strict-origin-when-cross-origin


Resopnse:

<!doctype html>
<html lang=en>
    <head>
        <title>http://localhost:1313/v1.0.2/tasks/observability/proxy-observability/</title>
        <link rel=canonical href=http://localhost:1313/v1.0.2/tasks/observability/proxy-observability/>
        <meta name=robots content="noindex">
        <meta charset=utf-8>
        <meta http-equiv=refresh content="0; url=http://localhost:1313/v1.0.2/tasks/observability/proxy-observability/">
    </head>
</html>

```

**Comment by zirain** [comment]
close this one, as it won't help.

**Pull request closed without merging**